app/models/annotation/nml/NmlParser.scala

```diff
@@ -26,6 +26,7 @@ object NmlParser extends LazyLogging with ProtoGeometryImplicits with ColorGener
   private val DEFAULT_RESOLUTION = 0
   private val DEFAULT_BITDEPTH = 0
   private val DEFAULT_DESCRIPTION = ""
+  private val DEFAULT_WKURL = ""
```

Comment:
I think I’d feel better if the wkurl would be an option rather than an emptystring default. I understand that you copied this from description, but even there I don’t feel good about it. Unfortunately, changing it for the description would be a bit more complicated (this goes all the way into postgres). So I’m a bit conflicted about this. What do you think? Would it be nicer to use Option[String]? Or does it seem cleaner to build it identical to description?

Reply:
No, option is better for sure.